src/Resources/ResourceManager/Properties/Resources.resx

```diff
@@ -495,7 +495,7 @@ You can help us improve the accuracy of the result by opening an issue here: htt
     <value>Unrecognized resource change {0}: {1}. Specify one ore more values in the following list and try again: {2}.</value>
   </data>
   <data name="BicepNotFound" xml:space="preserve">
-    <value>Cannot find Bicep. Please add Bicep to your PATH or visit https://github.com/Azure/bicep/releases to install Bicep.</value>
+    <value>Cannot find Bicep. Please add Bicep to your PATH or visit https://github.com/Azure/bicep/blob/main/docs/installing.md#install-the-bicep-cli to install Bicep.</value>
```

Comment:
I prefer to removing the anchor. 
I meant https://github.com/Azure/bicep/blob/main/docs/installing.md